### DISCUSSION TITLE
Request - path editable

### DISCUSSION THREAD

**Discussion by ldmanfred** [JohanFalt/Foundry_WoD20#556] [Q&A] [answered]
I am using this system and one of my players tries to have a path instead of humanity but the ones that appear cannot be found.
Could you put some option to put a path that you can put whatever you want?

pd: I take advantage of the request I made to congratulate you on the work on updates and constant attention to the system.

**Comment by ldmanfred** [comment]
Duplicate sry

**Comment by JohanFalt** [comment, marked as answer]
Is planned for v2.5.
